ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/ForeignKeyTableTableJoin.java

```diff
@@ -43,6 +44,8 @@ public ForeignKeyTableTableJoin(
       final JoinType joinType,
       @JsonProperty(value = "leftJoinColumnName", required = true)
       final ColumnName leftJoinColumnName,
+      @JsonProperty(value = "leftFormats", required = true)
```

Comment:
It's not important that these formats are from the left source, right? The purpose is to record the formats used as the output for this step. If this is true, let's rename this to `outputFormats` or `resultFormats` or something similar, to avoid confusion in the future.